composer.json

```diff
@@ -59,8 +53,23 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"check-all": [
```

Comment:
Should this be documented in https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/.github/CONTRIBUTING.md?

Reply:
@dingo-d If you like, I can add that to the doc updates commits I have in my WIP-3.0.0 branch ?

Reply:
Added!